### DISCUSSION TITLE
ForecastingGridSearchCV usage

### DISCUSSION THREAD

**Discussion by jpss95** [alan-turing-institute/sktime#4783] [Q&A] [answered]
I'm trying to do a ForecastingGridSearchCV but it keep giving me this error:

`Input contains NaN, infinity or a value too large for dtype('float64').
`

I've tried doing it with my data and with dummy values, either way I get the same error even though I'm sure the data doesn't have any NaNs, infinities or very large values. 

To reproduce the error please take in consideration this code:
https://gist.github.com/jpss95/79cf69a1e9060e8fe5459f64555422af

## Versions
sktime==0.18.0

# Dummy data to reproduce error
    #Generate date range from January 2016 to April 2023
    start_date = '2016-01'
    end_date = '2023-04'
    date_range = pd.date_range(start=start_date, end=end_date, freq='M')

    # Generate random demand data
    np.random.seed(123)  # Set a seed for reproducibility
    demands = np.random.randint(100, 1000, size=len(date_range))

    # Create the time series DataFrame
    y = pd.DataFrame({'Demand': demands}, index=date_range)

    y.index = pd.PeriodIndex(y.index, freq="M")`

**Comment by jpss95** [comment, marked as answer]
I just solved my problem by using a largest initial_window_size for the expanding window method

**Reply by fkiraly** [reply]
can you perhaps still paste your full code so we can learn about the issue?

(if it´s not too much of a hassle)
